app/posts/apply-for-teacher-training/2024-09-02-communicating-enic-to-international-candidates.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Helping international candidates show that their qualifications are equivalent to UK ones
+description: Communicating the certificate of comparability to international candidates so that they can apply for one at the right time in their application.
+date: 2024-09-02
+tags:
+    - ENIC
+    - international
+    - qualifications
+
+---
+
+## The problem we are trying to solve
+
+In the 2023 to 2024 recruitment cycle we saw a 269% increase in applications from international candidates compared to the previous cycle, but a 25.8% reduction in the success rate of these applications.
+
+Training providers reported that they were struggling to process the high number of applications because of the extra time required to check qualifications gained overseas to make sure that they match the entry requirements.
+
+ It can take providers 10 times longer to process applications from international candidates. (From user research in December 2023)
```

Comment:
```suggestion
It can take providers 10 times longer to process applications from international candidates (From user research in December 2023).
```